cli/op_error.rs

```diff
@@ -261,11 +261,11 @@ impl From<&ReadlineError> for OpError {
   fn from(error: &ReadlineError) -> Self {
     use ReadlineError::*;
     let kind = match error {
-      Io(err) => return err.into(),
+      Io(err) => return OpError::from(err),
```

Comment:
What changed? I don't understand how this is different from before.

Reply:
Nothing just sloppy editing